### DISCUSSION TITLE
Help running tests

### DISCUSSION THREAD

**Discussion by thugdoubt** [pycontribs/jira#1685] [Q&A]
Hello. I'm attempting to run the tests, but I'm unable to start the jira container as specified [here](https://github.com/pycontribs/jira/#manual). I'd appreciate some guidance with the following error:

```
Failed to execute goal com.atlassian.maven.plugins:jira-maven-plugin:8.0.2:run (default-cli) on project myartifact: Unable to find/resolve artifact.:
Could not transfer artifact com.atlassian.jira:atlassian-jira-webapp:war:8.9.0 from/to atlassian-public (https://maven.atlassian.com/repository/public):
GET request of: com/atlassian/jira/atlassian-jira-webapp/8.9.0/atlassian-jira-webapp-8.9.0.war from atlassian-public failed: Connection reset -> [Help 1]
```

**Comment by adehad** [comment]
Hard to say what the issue might be, I could suggest specifying a specific jira version, to match what we are [doing in the CI](https://github.com/pycontribs/jira/blob/6f58b704c08d12925f217b686ca1da57726fa120/.github/workflows/jira_server_ci.yml#L20):

```
docker run -dit -p 2990:2990 --name jira addono/jira-software-standalone --version 8.17.1
```

**Reply by thugdoubt** [reply]
Thanks for the response. I did try this, same result.

**Reply by adehad** [reply]
Could you have a look at using your browser to access: https://maven.atlassian.com/repository/public

Perhaps there is some firewall or otherwise something preventing you from accessing the resources needed by the docker image?

**Reply by thugdoubt** [reply]
I am able to access that repository, can confirm the expected path is present and I'm able to download the .war file. I am able to fetch the war file from a running Docker container. :(

**Reply by adehad** [reply]
Wow that is strange, sorry I can't be of more help.

I know waiting for the CI can be a bit tedious, perhaps you can look into using a service like [railway.app](railway.app) or [fly.io](fly.io) to deploy the Dockerfile ? I think their free tiers should be sufficient.
